GitOut/Features/Git/RepositoryList/RepositoryListViewModel.cs

```diff
@@ -81,6 +76,59 @@ ISnackbarService snack
             });
 
             ClearCommand = new CallbackCommand(() => SearchQuery = string.Empty);
+
+            DropCommand = new AsyncCallbackCommand<System.Windows.DataObject>(OnDropAsync);
```

Comment:
Could use a using instead

Reply:
we're using System.Windows.Forms which also has a DataObject. So then we'd have to differentiate them
would `using DataObject = System.Windows.DataObject;` be better?

Reply:
I think so, yes. Both would be ok with me, didn't consider `DataObject` being available in another namespace.
Feel free to resolve this comment when you change this.